pom.xml

```diff
@@ -69,11 +69,11 @@
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<maven.compiler.release>8</maven.compiler.release>
 		<scala.minor.version>2.11</scala.minor.version>
-		<scala.version>${scala.minor.version}.11</scala.version>
```

Comment:
Ok for me to increase these version before our 1.0.0 release. Otherwise we shouldn't always change our dependencies...